core/src/repair/ancestor_hashes_service.rs

```diff
@@ -784,26 +784,16 @@ impl AncestorHashesService {
     ) {
         dead_slot_pool.retain(|dead_slot| {
             let epoch = root_bank.get_epoch_and_slot_index(*dead_slot).0;
-            if let Some(epoch_stakes) = root_bank.epoch_stakes(epoch) {
+            //TODO: figure out if we even need to make this check
+            if let Some(_epoch_stakes) = root_bank.epoch_stakes(epoch) {
                 let status = cluster_slots.lookup(*dead_slot);
-                if let Some(completed_dead_slot_pubkeys) = status {
-                    let total_stake = epoch_stakes.total_stake();
-                    let node_id_to_vote_accounts = epoch_stakes.node_id_to_vote_accounts();
-                    let total_completed_slot_stake: u64 = completed_dead_slot_pubkeys
```

Comment:
We no longer need to compute this here as clusterslots has this information cached for us